### ISSUE TITLE
change obi stop to send SIGTERM instead of SIGINT

### ISSUE THREAD

**Issue by Gobiner** [Oblong/obi#85]
https://github.com/Oblong/obi/issues/80#issuecomment-272594022 https://github.com/Oblong/obi/pull/79#discussion_r96128593

**Comment by Gobiner** [comment]
Currrently you cannot stop a run of `obi go hex --debug=gdb` with `obi stop`.
```
oblong@hex-b-1:~$ pgrep -fl '/tmp/([a-z_][a-z0-9_]{0,30})/newsdisco/build/newsdisco '
25605 tmux
25606 gdb
oblong@hex-b-1:~$ pkill -SIGINT -f '/tmp/([a-z_][a-z0-9_]{0,30})/newsdisco/build/newsdisco '
oblong@hex-b-1:~$ pgrep -fl '/tmp/([a-z_][a-z0-9_]{0,30})/newsdisco/build/newsdisco '
25605 tmux
25606 gdb
oblong@hex-b-1:~$ pkill -SIGTERM -f '/tmp/([a-z_][a-z0-9_]{0,30})/newsdisco/build/newsdisco '
oblong@hex-b-1:~$ pgrep -fl '/tmp/([a-z_][a-z0-9_]{0,30})/newsdisco/build/newsdisco '
oblong@hex-b-1:~$
```

**Comment by jshrake** [comment]
Closed by 7766d92f42

We can track #86 in #80
